src/wizards/tokenSwapDealWizard/components/daoStageContent/daoStageContent.ts

```diff
@@ -35,4 +38,28 @@ export class DaoStageContent {
     this.data.social_medias.splice(index, 1);
     this.form.revalidateErrors();
   }
+
+  attached() {
+    this.hydrateDaosList();
+  }
+
+  private prepareAvatarUrl(url) {
```

Comment:
We could do this inside the firebase function when we fetch the list of DAOs from DeepDAO :)